integreat_cms/cms/utils/stringify_list.py

```diff
@@ -3,23 +3,30 @@
 from django.utils.translation import gettext_lazy as _
 
 
-def iter_to_string(iterable: Iterable, *, quotation_char: str = '"') -> str:
+def iter_to_string(
+    iterable: Iterable, *, quotation_char: str = '"', max_items: int = 5
+) -> str:
     """
     This is a helper function that turns a list into a string using different delimiters.
 
     :param iterable: List of items
     :param quotation_char: The character that is added around each item in the list
+    :param max_items: The maximum number of items in the iterable to display
     :return: Returns string with the single items separated by comma expect the last item which is separated by 'and'.
     """
     # Convert iterable to list to support querysets etc.
     lst = list(iterable)
-    # If the list contains more than 1 element, save the last element for later
-    last_element = lst.pop() if len(lst) > 1 else None
+    last_element = lst.pop() if 1 < len(lst) <= max_items else None
     # Join remaining elements with commas and surrounding quotes
     separator = quotation_char + ", " + quotation_char
-    list_str = quotation_char + separator.join(map(str, lst)) + quotation_char
+    list_str = (
+        quotation_char + separator.join(map(str, lst[:max_items])) + quotation_char
+    )
     # Append the last element with "and"
     if last_element:
         list_str += " " + _("and") + f" {quotation_char}{last_element}{quotation_char}"
-    # Return final string
+    elif len(lst) - max_items == 1:
```

Comment:
This might be controversial but in order to make the code a bit more readable I would prefer to put this conditional into a variable like:

only_one_more_element_left = len(lst) - max_items == 1

and then use it here as

```suggestion
    elif only_one_more_element_left:
```
In my option this improves the code a little. But since I couldn't find a clear guideline on this, I'd also be okay if you decide to reject this suggestion :)

If you decide to follow the suggestion, I'd like it if you could also adapt something like this for the line below

Reply:
I went ahead and added a variable `number_elements_over_limit` and also renamed the variable `last_element` to make it more descriptive. Could you take a look at the changes? :)

Reply:
Looks good! I like the changes. This is better than what I had suggested. I only have one question left, which you can find below :)